web_app/analisis/views.py

```diff
@@ -63,17 +63,38 @@ def list(self, request):
         serializer = AnswerSerializer(queryset, many=True)
         return Response(serializer.data)
 
+
+def add_id_to_in_progress_ids(_id):    
+    from django.core.cache import cache
+    cache.set('IN_PROGRESS_IDS', [_id] + cache.get('IN_PROGRESS_IDS',[]), timeout=36000)# 1 hour
```

Comment:
Could we parametrize these functions to something like `add_id_to_in_progress_ids(ids_type, _id)`, `remove_id_from_in_progress_ids(ids_type, input_id)` , ` get_in_progress_ids(ids_type)`

And in calls something like:
- `add_id_to_in_progress_ids('PURE_ANNOTATION', _id)`
- `remove_id_from_in_progress_ids('PURE_ANNOTATION', input_id)`
- `get_in_progress_ids('PURE_ANNOTATION')`

This is because we will use same functionality for resolving tweets and then we could use same functions only with a different key name, example: `RESOLVE_TWEET_RELATION`

Reply:
Added new param, rename functions and refactor.